digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/repository/SyncRepository.java

```diff
@@ -0,0 +1,29 @@
+package org.digit.health.sync.repository;
+
+import org.digit.health.sync.web.models.dao.SyncData;
+import org.digit.health.sync.web.models.request.SyncSearchDto;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class SyncRepository  {
```

Comment:
Make SyncLogRepository an interface and create an implementation called DefaultSyncLogRepository